apps/server/src/modules/lesson/controller/dto/task.response.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { InputFormat } from '@shared/domain/types';
+
+export class SlimTaskResponse {

```

Comment:
maybe the following name describes the response better? 
```suggestion
export class SlimTaskResponse {
```

Reply:
Done